test/runtime.js

```diff
@@ -173,22 +206,44 @@ describe('runtime parameters', function() {
         assert.equal(process.sass.binaryUrl.substr(0, URL.length), URL);
       });
     });
-    describe('checking if environment variable overrides package.json', function() {
+    describe('checking if environment variable overrides .npmrc configuration variable', function() {
```

Comment:
I tried to avoid redundancy there. The test is still existing but tests that .npmrc parameters can override package.json parameters ( see `checking if .npmrc configuration variable overrides package.json`). And `checking if environment variable overrides .npmrc configuration variable` checks if env parameters can override .npmrc parameters. This covers the whole chain form a hierarchical point of view. 

I can of course add the original tests again if you like.